docs/en/ingest-management/commands.asciidoc

```diff
@@ -37,10 +37,7 @@ invoking the wrong binary.
 == elastic-agent diagnostics
 
 Gather diagnostics information from the {agent} and component/unit it's running.
-
-If no options are specified, this command displays version numbers and component/unit metadata.
-
-If `collect` is specified, it produces an archive containing:
+It produces an archive containing for the Elastic Agent:
```

Comment:
```suggestion
This command produces an archive that contains:
```
Edited for a couple of reasons: "it" has an unclear antecedent. Also "...containing for the Elastic Agent" is a bit awkward grammatically and IMO is unnecessary because it's redundant (already stated in the first sentence).